Changes

```diff
@@ -190,6 +190,9 @@ _______________
   (Sébastien Hinderer and Stephen Dolan, review by Damien Doligez and
   Hugo Heuzard)
 
+- #13326: Implement Unix.O_APPEND on windows.
+  (Romain Beauxis)
```

Comment:
You should indicate the reviewers (Miod Vallat, Gabriel Scherer, Antonin Décimo).

Reply:
Done!